bedrock/mozorg/templates/mozorg/home/includes/ctd-promo-de.html

```diff
@@ -0,0 +1,61 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% block content %}
+  <section class="ctd-promo">
+
+    <!-- CTD Promo for non-Firefox users -->
+    <div class="ctd-promo-non-fx">
+      {% call split(
+        block_class='mzp-t-split-nospace mzp-l-split-center-on-sm-md t-pocket-promo',
+        theme_class='mzp-c-split-bg',
```

Comment:
`theme_class` is only needed if there's a full-bleed background. If the parameter is present the macro automatically adds the wrapper element with `mzp-c-split-bg` so this parameter's value should be for some other theming, either one of the standard background classes or a custom one.

Since there's no background on this promo you can remove the parameter, but then you should also remove the `mzp-t-split-nospace` class from the outer block so the spacing is correct.